Gemfile

```diff
@@ -1,7 +1,6 @@
-# encoding: utf-8
-
 source 'https://rubygems.org'
 
 gemspec name: 'mutant'
 
 gem 'devtools', git: 'https://github.com/rom-rb/devtools.git'
+eval_gemfile File.expand_path('../Gemfile.shared', __FILE__)
```

Comment:
The `Gemfile.shared` file is empty. What is the reason for evaling it? Why not just remove the file and this line.